src/erlcloud_mon.erl

```diff
@@ -216,33 +217,111 @@ put_metric_data(Namespace, MetricName, Value, Unit, Timestamp) ->
          ),
     mon_simple_query(Config, "PutMetricData", Params).
 
+
+%%------------------------------------------------------------------------------
+%% @doc CloudWatch API - GetMetricStatistics - Easy average version
+%% Gets average stats at 60 second intervals for the given metric on the given instance for the given interval
+%% @end
+%%------------------------------------------------------------------------------
+-spec get_metric_statistics(
+        MetricName  ::string(),
+        StartTime   ::datetime(),
```

Comment:
It looks like these can be `datetime() | string()`, as the local module's `format_datetime/1` function has a head for both the datetime tuple and a string. So as long as you pass an AWS-compliant datetime string, that works too. Probably worth noting in the specs for `StartTime`/`EndTime`.